api/src/pcapi/core/subscription/ubble/api.py

```diff
@@ -233,7 +234,7 @@ def _create_ubble_identification(
 def _update_identity_fraud_check(
     fraud_check: fraud_models.BeneficiaryFraudCheck, content: fraud_models.UbbleContent
 ) -> None:
-    fraud_check.thirdPartyId = content.identification_id
+    fraud_check.thirdPartyId = typing.cast(str, content.identification_id)
```

Comment:
```suggestion
    fraud_check.thirdPartyId = content.identification_id or ""
```
cette histoire de `cast` n'a pas l'air de fonctionner sur `None`